pkg/xray/resource_xray_ignore_rule.go

```diff
@@ -28,7 +28,7 @@ type IgnoreRule struct {
 
 type IgnoreFilters struct {
 	Vulnerabilities  []string                      `json:"vulnerabilities,omitempty"`
-	Licenese         []string                      `json:"licenses,omitempty"`
+	Liceneses        []string                      `json:"licenses,omitempty"`
```

Comment:
oh man, this has a typo, should be `Licenses` not `Liceneses`

Reply:
Good 👁️